cypress/integration/search.spec.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+// Tests for search and advanced search
+
+describe("Search", () => {
+    it("should display the search results", () => { 
+        cy.visit(Cypress.env("home"))
+
+        cy.get('a[name="word-link"]:first')
+            .then((word) => {
+                const w = word[0].id
```

Comment:
Minor note on naming. If I understand this correctly, the thing yielded by `cy.get('a[name="word-link"]:first')` is a `NodeList`. So maybe its name should be something like `anchorElements`? And then the thing you get from it is the `word`. i.e.,

```suggestion
            .then((anchorElements) => {
                const word = anchorElements[0].id
```